.cypilot/requirements/readme-checklist.md

```
@@ -0,0 +1,696 @@
+1. [Agent Instructions](#agent-instructions)
+2. [Review Scope Selection](#review-scope-selection)
+3. [Prerequisites](#prerequisites)
+4. [Applicability Context](#applicability-context)
+5. [Severity Dictionary](#severity-dictionary)
+6. [MUST HAVE](#must-have)
+   - [Identity & Purpose (IDENT)](#-identity--purpose-ident)
+   - [Getting Started (START)](#-getting-started-start)
+   - [Usage Documentation (USAGE)](#-usage-documentation-usage)
+   - [Technical Information (TECH)](#-technical-information-tech)
+   - [Community & Contribution (COMM)](#-community--contribution-comm)
+   - [Maintenance & Quality (MAINT)](#-maintenance--quality-maint)
```

markdownlint-cli2 (0.21.0)

[warning] Link fragments should be valid

(MD051, link-fragments)